name.abuchen.portfolio.bootstrap/OSGI-INF/l10n/bundle.properties

```diff
@@ -36,6 +36,8 @@ command.openbrowser.changelog.name           = Changelog
 command.openbrowser.changelog.tooltip        = https://github.com/buchen/portfolio/releases
 command.openbrowser.faq.tooltip              = https://forum.portfolio-performance.info/c/faq
 command.openbrowser.forum.tooltip            = https://forum.portfolio-performance.info/c/english/
+command.openbrowser.gitHubTeam.name          = Join development team
```

Comment:
In contrast to the link to POEditor, this is not directly joining the development team. I'll rephrase it more generically to "Source Code on Github"

Reply:
> "Source Code on Github"

Maybe "Project on Github", because my understanding that the idea is to advertise place we people can get together to try to improve the software, not just grab the source code...

Reply:
People who know GitHub (approximately 100% of developers) will already know that it is more than just a download site. For other users, who may just be curious, it is more helpful to mention one important example of what they can find here; for general discussion and suggestions to improve PP, they should first and foremost come to the forums, not here.